docs/feature-report/README.md

```diff
@@ -0,0 +1,553 @@
+in 2021.
+
+TCS 2, which we now bring to public review, converts the TCS 1 XML Schema to a vocabulary standard, like Darwin Core, that can be maintained under the VMS.
+
+## Parameters
+
+The purpose of the current work is to make TCS useable again by changing it from
+an XML Schema to a vocabulary standard. Changes we propose are as far as
+possible only structural and do not affect the meaning of terms. Because TCS 2
+is a vocabulary standard we have had to provide new definitions for many of the
+terms, as in TCS 1 the meaning of elements and attributes was largely implicit,
+but the Taxon Concept and Taxon Name in TCS 2 are the Taxon Concept and Taxon
+Name in TCS 1 and the same goes for all other terms. No completely new terms
+have been added.
+
+Where we can we borrow terms from other standards, rather than defining them in
+TCS. The parsed name properties, as well as some other Taxon Name properties are
+borrowed from Darwin Core and some Taxon Concept properties from Dublin Core.
+There are some other terms which have been defined in TCS as IRI properties,
+which have literal equivalents in Darwin Core. In most cases, they will share
+the same label.
+
+While the version of the standard we are replacing is an XML Schema (XSD), we
+took a lot from the TDWG Taxon Concept and Taxon Name LSID Ontologies, which
+form a rather precise translation of the TCS XML Schema into OWL ontologies.
+Because the TDWG Ontologies never made it into a standard, we could not borrow
+any terms from them, but a lot from the TDWG Ontologies can be seen in TCS 2.
+
+## Changes
+
+The most important change is that while in TCS 1 the `Name` element on of the
+Taxon Concept is required and the `AccordingTo` element is not, in TCS 2 it is
+the `accordingTo` property that is required. Every TCS 2 Taxon Concept object
+has to have a source, or `accordingTo`, which is a publication or other other
+form of communication where a taxon is defined in a certain way. This means that
+every Taxon Concept is traceable to a source and identifiable, and that in TCS 2
+nominal concepts are not possible. With regards to the Name, in TCS 1 a `Name`
+can either be a text node, so a string, or a reference (`ref`) to a `TaxonName`
+element with an `id` attribute. This is completely analogous to the use of the
+Darwin Core `scientificName` and the TCS `taxonName`, respectively, in TCS 2.
+
+The most important structural change we made was dismantling the Taxon
+Relationship object that both TCS 1 and the Taxon Concept Ontology have. We want
+TCS to be a vocabulary standard, i.e. a set of terms and definitions, so TCS
+should not prescribe a certain syntax. 
+
+Another problem with a relationship object is that it obscures the precise
+nature of the relationship. Not all relationship types in TCS 1 are
+relationships between Taxon Concepts, some are relationships between Taxon
+Concepts and Taxon Names. Also, relationships between Taxon Names in TCS 1 are
```

Comment:
I must have made another change in this part, so I could not just commit this suggestion, but I will work it in manually.

`HasSynonym` is a bit of an unfortunate example, as this is a mixed concept in TCS 1, consisting of synonyms in the strict sense as well as things that are more like the set relationships. I like to think that we split the TCS 1 `has synonym` into `synonym` and `intersects`, so that we can say that `intersects` is not an addition, even though `intersects` can be applied more broadly than `has synonym` (or than I initially thought).

Also, while 'synonym' is indeed a name-to-name relationship, `synonym` in TCS 2 is a concept-to-name relationship, just like `taxonName`